src/utils/transaction.ts

```diff
@@ -43,6 +42,22 @@ import { ITransactionBlock } from '../types/transactionBlock';
 import { PLR_STAKING_ADDRESS_ETHEREUM_MAINNET, POLYGON_USDC_CONTRACT_ADDRESS } from '../constants/assetConstants';
 import { PlrV2StakingContract } from '../types/etherspotContracts';
 
+interface IPillarDAO {
+  encodeDeposit(amount: BigNumber): {
+    to: string;
+    data: string;
+  };
+};
+
+interface IPLRTransaction {
```

Comment:
This is a very minor nitpick, but could we stick to camel case even with symbol names 
```suggestion
interface IPlrTransaction {
```